terasoluna-gfw-common-libraries/terasoluna-gfw-security-web/src/test/resources/logback.xml

```diff
@@ -5,14 +5,6 @@
       <pattern><![CDATA[date:%d{yyyy-MM-dd HH:mm:ss}\tthread:%thread\tlevel:%-5level\tlogger:%-48logger{48}\tmessage:%replace(%msg){'(\r\n|\r|\n)','$1  '}%n%replace(%replace(%xEx){'(\r\n|\r|\n)','$1  '}){'  $',''}%nopex]]></pattern>
     </encoder>
   </appender>
-  <appender name="DB" class="ch.qos.logback.classic.db.DBAppender">
```

Comment:
- [ ] Since you deleted the appender "DB", isn't the following unnecessary?
  https://github.com/terasolunaorg/terasoluna-gfw/blob/37cb0a8ed32d4f397283790f6b64c938df093c03/terasoluna-gfw-common-libraries/terasoluna-gfw-security-web/src/test/resources/logback.xml#L13
  
  This too.
  https://github.com/terasolunaorg/terasoluna-gfw/blob/37cb0a8ed32d4f397283790f6b64c938df093c03/terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/test/resources/logback.xml#L12

Reply:
Delete unnecessary lines
0716443